chemprop/v2/tests/unit/featurizers/test_atom.py

```diff
@@ -85,7 +85,7 @@ def test_len(featurizer, expected_len):
 
 
 def test_num_subfeatures():
-    assert AtomFeaturizer().num_subfeatures == 8
+    assert len(AtomFeaturizer()._subfeats) == 6
 
```

Comment:
Note there are indeed eight subfeatures (see the docstring). And I think it is not intended for users to call ` len(AtomFeaturizer()._subfeats)` to get the number of subfeatures.

Instead of changing this test, I suggest adding the `num_subfeatures` method and letting it return len(self._subfeats) + 2.

Reply:
no need to test this attribute as it's not publicly exposed. Internals may change in the future and we don't want our test to assume structure about internals

Reply:
Thank you for explaining. The test has been removed as suggested.